man/adoc/bpftrace.adoc

```diff
@@ -1285,6 +1290,37 @@ END {
 }
 ----
 
+[#probes-bench]
+=== BENCH
+
+`BENCH` is a special built-in probe type for creating microbenchmarks.
+bpftrace executes each `BENCH` probe repeatedly to measure the average
+execution time of the contained code. If multiple `BENCH` probes exist
+in a script, bpftrace executes them sequentially in the order they are
+specified. `BENCH` probes execute before `BEGIN`.
+
+----
+BENCH:lhist {
+    @a = lhist(rand % 10, 1, 10, 1);
+}
+
+BENCH:count {
```

Comment:
Maybe it's good to add an example that doesn't use one of bpftrace's predefined names so users don't think it's a limitation e.g. just change `BENCH:count` to `BENCH:my_count` or something.

Reply:
Sure, will do.

Reply:
Done.